src/Microsoft.Net.Http.Headers/CookieHeaderValue.cs

```diff
@@ -39,7 +39,7 @@ public string Name
             get { return _name; }
             set
             {
-                CheckNameFormat(value, "name");
```

Comment:
@hishamco the majority of the original code changed in this PR (using `"value"`) was correct. Should not `throw` `Exceptions` containing names of backing fields or other private implementation details. E.g. this one should have been `"value"` and now `nameof(value)`.